### ISSUE TITLE
Question on experiment settings

### ISSUE THREAD

**Issue by vincentlux** [Computer-Vision-in-the-Wild/DataDownload#2]
Hi, thanks a lot for organizing this benchmark dataset! Can you help clarify two details of the OD benchmark?

1. For the eval scores reported in the ELEVATER paper, what is the evaluation metric? is it mAP or mAP@50?
2. For the eval scores reported in the ELEVATER paper, are these scores from the test set or the validation set? Does [this line](https://github.com/Computer-Vision-in-the-Wild/DataDownload/blob/master/detection/odinw_35/AerialMaritimeDrone_large.yaml#L93 ) indicate that the number is from the validation set?

Thanks a lot for your help!

**Comment by liunian-harold-li** [comment]
Hi, thank you for your interest in our benchmark

1. It's mAP@50:95 as in COCO evaluation.
2. We used test set for final reporting. The config was for fine-tuning. For evaluation, we changed the TEST to "test" in the eval code.

**Comment by vincentlux** [comment]
Got it! Thanks for your quick reponse!
